cla_public/static-src/javascripts/modules/find-legal-adviser.js

```diff
@@ -307,3 +307,18 @@
     }
   };
 
+  function setMapHeight(setHeight) {
+    if ($(window).width() > 640) {
+      if ($("ul.org-list").height() > setHeight) setHeight = $("ul.org-list").height();
+      $(".search-results-container").height(setHeight);
+      $(".search-results-list").height(setHeight);
```

Comment:
I think these two lines could be combined, using both selectors in the lookup

Reply:
Done